transformer/cloudformation/resources.tf

```diff
@@ -0,0 +1,31 @@
+resource "null_resource" "get_stack_names" {
+  provisioner "local-exec" {
+    command = <<-EOT
+      aws cloudformation describe-stacks --query 'Stacks[*].{Name:StackName}' --output json > stack_names.json
+    EOT
+  }
+
+  # Trigger the provisioner only once
+  triggers = {
+    always_run = "${timestamp()}"
+  }
+}
+
+
+resource "aws_s3_object" "upload_templates" {
+  for_each = data.aws_cloudformation_stack.resource
+
+  bucket = var.s3Bucket
+  key    = "${var.s3_path}/${each.key}.yaml"
+  content = each.value.template_body
```

Comment:
User might want to pass encryption info as well. Please create a task for working on it later.